exporter.go

```diff
@@ -81,16 +92,15 @@ func execute(name string, c collector.Collector, ch chan<- prometheus.Metric) {
 	begin := time.Now()
 	err := c.Collect(ch)
 	duration := time.Since(begin)
-	var result string
 
 	if err != nil {
 		log.Errorf("ERROR: %s collector failed after %fs: %s", name, duration.Seconds(), err)
-		result = "error"
+		scrapeSuccess.WithLabelValues(name).Set(0)
```

Comment:
This should use ConstMetrics, so there's no interference from any scrapes happening at the same time